src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/services/UploadEnvelopeDocumentsService.java

```diff
@@ -185,7 +181,7 @@ private void uploadParsedZipFileName(Envelope envelope, List<Pdf> pdfs) {
             documentProcessor.uploadPdfFiles(pdfs, envelope.getScannableItems());
 
             log.info(
-                "Uploaded pdfs. File {}, Container: {}, Envelope ID: {}",
+                "Uploaded PDF files to Document Management. File {}, Container: {}, Envelope ID: {}",
```

Comment:
This change is to make with message use the same language as the corresponding message in exception handling.